src/arith/iter_affine_map.cc

```diff
@@ -160,13 +160,22 @@ class IterMarkSplitCollector {
   }
 };
 
+/*! \brief Record form of IterMark(x, extent) + offset */
+struct IterMarkWithOffset {
```

Comment:
Is it necessary to introduce offset, since IterMark can contain an IterSumExpr, and IterSumExpr can already represent iter + base? Maybe you can consider utilizing this.

For example, `i*8 + j*2 + k` with `1 <= j * 2 + k <= 9, i <= 10`.
The flattened form is `i*8 + j*2 + k`.
The structured form is `i*8 + (j*2 + k - 1) + 1`
The iter mark contains the structured form and extent 80. We can read from the structured form that the offset it 1.

Reply:
OK I now think it is necessary, since we need to note down the offset modified by the lower bound constraint.